_rules/links-with-identical-names-serve-equivalent-purpose-b20e66.md

```diff
@@ -24,21 +25,26 @@ htmlHintIgnore:
 
 ## Applicability
 
-This rule applies to any set of any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of `link`, or a role that inherits from the `link` role, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-characters) [accessible names][accessible name] that do not only consist of [whitespace](#whitespace).
+This rule applies to any set of any two or more HTML or SVG elements which
 
-**Note:** The test target for this rule is the full set of link elements that share the same [matching](#matching-characters) [accessible name][].
+- have the [semantic role][] of `link`, or a role that inherits from the `link` role; and
+- are in the same [set of browsing context related nodes][]; and
+- are [included in the accessibility tree][] of their respective [document][]; and
+- that have [matching][] [accessible names][accessible name] that do not only consist of [whitespace](#whitespace).
+
+**Note:** The test target for this rule is the full set of link elements that share the same [matching][] [accessible name][].
 
 ## Expectation
 
-When followed, the links in each set of target elements resolve to the [same resource](#same-resource) or to [equivalent resources](#equivalent-resource). Resolving the links includes potential redirects, if the redirects happen instantly.
+When followed, the links in each set of target elements resolve to the [same resource][] or to [equivalent resources](#equivalent-resource). Resolving the links includes potential redirects, if the redirects happen instantly.
 
 ## Assumptions
 
 This rule assumes that the purpose of the links with identical [accessible names][accessible name] would not be ambiguous to users in general when seen in context on the web page, which is the exception mentioned in success criterion [2.4.9 Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html). If the links are ambiguous to users in general, users of assistive technologies are not at a disadvantage when viewing the links out of context, e.g. on a list of links in a screen reader, which makes it more of a general user experience concern than an accessibility issue.
 
 ## Accessibility Support
 
-There are no major accessibility support issues known for this rule.
+This rule assume that assistive technologies are exposing all links on the page in the same way no matter which [document tree](https://dom.spec.whatwg.org/#document-trees) they are in. If an assistive technology requires the user to "enter" an `iframe` or a [shadow tree][] before exposing its links, then it is possible for two links to have identical name but resolve to different resources without breaking success criterion [2.4.9 Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html) (if said links are in separate [documents][document] or [shadow trees][shadow tree])
 
 ## Background
```

Comment:
Because you're using shadow tree, which is in a draft spec, can you add a link to the draft spec in the background?